components/formats-gpl/src/loci/formats/in/NativeND2Reader.java

```diff
@@ -747,13 +747,14 @@ else if (blockType.startsWith("ImageText")) {
           long startFP = in.getFilePointer();
           in.seek(startFP - 1);
 
-          String textString = DataTools.stripString(in.readString((int) dataLength));
-          textStrings.add(textString);
-          validDimensions.add(blockCount > 2);
+          int typeByte = in.read();
+          int charCount = in.read();
+          textStrings.add(DataTools.stripString(in.readString(charCount * 2)));
+          int numTextInfos = in.readInt();
+          long remainingBytes = in.readLong();
 
-          if (!textString.startsWith("<")) {
-            skip = 0;
-          }
+          iterateIn(in, startFP + dataLength - 1);
```

Comment:
PR has caused some failures in the repo tests: https://merge-ci.openmicroscopy.org/jenkins/job/BIOFORMATS-test-folder/lastFailedBuild/console

I think dataLength here might need updated based on the values already read (so  - 14 + (charCount * 2))

It also seems that the value of skip might need reset to 0 as before

Reply:
c4a1a39 seems to fix this when testing locally, but I'll check the repo test tomorrow.  Certainly feel free to exclude if too disruptive right now.